engines/stark/ui/dialogbox.cpp

```diff
@@ -228,12 +228,7 @@ Graphics::Surface *DialogBox::loadBackground() {
 	// GOG version of The Longest Journey is 1.0.0.142 "RC1" (Special Build: "Paper Sun")
 	// Steam's game.exe does not contain a valid resource for the background bitmap id 147
 	// so we skip trying to retrieve it based on build number.
```

Comment:
The `based on build number` part of the comment should probably be removed?

Reply:
Indeed. I removed it now.